docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md

```diff
@@ -4,15 +4,17 @@ title: Monitoring Architecture
 
 ## End-to-end monitoring flow
 
-The complete monitoring flow in Kyma comes down to these components and steps:
+The monitoring flow in Kyma comes down to the following components and steps:
 
 ![End-to-end monitoring flow](./assets/obsv-monitoring-flow.svg)
```

Comment:
The text on the arrow from operator to alertmanager is missing. It should be the same text as on the other arrow

Reply:
Thanks for spotting it! Done